internal/network/command_private_link_attachment_delete.go

```diff
@@ -0,0 +1,63 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/deletion"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/resource"
+	"github.com/confluentinc/cli/v3/pkg/utils"
+)
+
+func (c *command) newPrivateLinkAttachmentDeleteCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "delete <id-1> [id-2] ... [id-n]",
+		Short:             "Delete one or more private link attachments.",
+		Args:              cobra.MinimumNArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validPrivateLinkAttachmentArgsMultiple),
+		RunE:              c.privateLinkAttachmentDelete,
+	}
+
+	pcmd.AddForceFlag(cmd)
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+
+	return cmd
+}
+
+func (c *command) privateLinkAttachmentDelete(cmd *cobra.Command, args []string) error {
+	environmentId, err := c.Context.EnvironmentId()
+	if err != nil {
+		return err
+	}
+
+	existenceFunc := func(id string) bool {
+		_, err := c.V2Client.GetPrivateLinkAttachment(environmentId, id)
+		return err == nil
+	}
+
+	if err := deletion.ValidateAndConfirmDeletionYesNo(cmd, args, existenceFunc, resource.PrivateLinkAttachment); err != nil {
+		return err
+	}
+
+	deleteFunc := func(id string) error {
+		if err := c.V2Client.DeletePrivateLinkAttachment(environmentId, id); err != nil {
+			return errors.Errorf(errors.DeleteResourceErrorMsg, resource.PrivateLinkAttachment, id, err)
+		}
+		return nil
+	}
+
+	deletedIds, err := deletion.DeleteWithoutMessage(args, deleteFunc)
+	deleteMsg := "Requested to delete %s %s.\n"
```

Comment:
We already have `errors.RequestedDeleteResourceMsg`

Reply:
nice, i will update other files as well

Reply:
@brianstrauch it looks like the `errors.RequestedDeleteResourceMsg` [here](https://github.com/confluentinc/cli/blob/main/pkg/errors/strings.go#L115) is `"Requested to delete %s \"%s\".\n"` which is not compatible with `utils.ArrayToCommaDelimitedString(deletedIds, "and")` [here](https://github.com/confluentinc/cli/blob/ab8604ce15331fa582cd944ec3eadacd9740ae96/pkg/utils/utils.go#L104).

There are extra double quotes in the output.

Reply:
Good point, let's use a custom format string!